va/va.h

```diff
@@ -820,6 +820,13 @@ typedef struct _VAConfigAttrib {
  *  simultaneously. And BRC would adjust accordingly. This is so called
  *  Parallel BRC. */
 #define VA_RC_PARALLEL                  0x00000200
+/** \brief Quality defined VBR
+ * Use ICQ to determine the good enough QP for each MB such that
+ * good enough quality can be obtained without waste of bits
+ * for this BRC mode, you must set all legacy VBR parameters
+ * and ICQ_quality_factor in \c VAEncMiscParameterRateControl
+ * */
+#define VA_RC_QVBR                      0x00000400
 
```

Comment:
In QVBR mode to setting ICQ_quality_factor is the weirdest thing, I think

Reply:
I prefer to rename ICQ_quality_factor to quality_factor.

Reply:
rename is not allowed for compatibility

Reply:
how about add some description for reusing the definition